src/src/rx-serial/SerialMavlink.cpp

```diff
@@ -91,8 +93,8 @@ uint32_t SerialMavlink::sendRCFrame(bool frameAvailable, bool frameMissed, uint3
     mavlink_msg_rc_channels_override_encode(this_system_id, this_component_id, &msg, &rc_override);
     uint16_t len = mavlink_msg_to_send_buffer(buf, &msg);
     _outputPort->write(buf, len);
-    
-    return DURATION_IMMEDIATELY;
+
+    return MIN_PACKET_INTERVAL;
```

Comment:
This method of rate limiting causes major impacts to TQTY on ESP8285 receivers. Need to use a "simpler" approach to check if we should forward the pkt out or not.